iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -405,6 +405,7 @@ def create_section(self, row):
             has_unpublished_changes=row['has_unpublished_changes'],
             search_description=row['search_description'],
             seo_title=row['seo_title'],
+            background_color=self.get_color_hex(row['extra_style_hints'])
```

Comment:
Could this be text color rather than background color? The color in v1 only affects the text, so we should be consistent here.

Reply:
done